Modules/System/Azure Blob Services API/src/ABSClientImpl.Codeunit.al

```diff
@@ -116,11 +116,16 @@ codeunit 9051 "ABS Client Impl."
     procedure PutBlobBlockBlobUI(OptionalParameters: Codeunit "ABS Optional Parameters"): Codeunit "ABS Operation Response"
     var
         OperationResponse: Codeunit "ABS Operation Response";
+        TempBlob: Codeunit "Temp Blob";
         Filename: Text;
         SourceStream: InStream;
+        ChooseFileTitleMsg: Label 'Choose the file to upload.';
     begin
-        if UploadIntoStream('*.*', SourceStream) then
+        Filename := FileManagement.BLOBImportWithFilter(TempBlob, ChooseFileTitleMsg, '', 'All Files (*.*)|*.*', '*.*');
```

Comment:
Where does `FileManagement` come from?

Reply:
Ah, you're 100% right about the File Management codeunit being in the base application. I did my testing in a separate extension. Makes sense it doesn't work. Excuse me!

I assume the fix for the UploadIntoStream function is in a different branch? Because it appears to be `if UploadIntoStream('*.*', SourceStream) then` in the master branch:

https://github.com/microsoft/ALAppExtensions/blob/main/Modules/System/Azure%20Blob%20Services%20API/src/ABSClientImpl.Codeunit.al#L122